types/react-native-fbsdk/index.d.ts

```diff
@@ -0,0 +1,1090 @@
+// Type definitions for react-native-fbsdk 0.6
+// Project: https://github.com/facebook/react-native-fbsdk
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { ComponentClass, Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+// tslint:disable-next-line:export-just-namespace
+export = FBSDK;
+export as namespace FBSDK;
```

Comment:
I used the same shape as the [React](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts) library definition, which seemed to be the only way I could get it to work in my code. 

Is there a better way?